PKGBUILD

```diff
@@ -6,15 +6,17 @@
 # Contributor: Victor Hugo Souza <vhbsouza@gmail.com>
 
 pkgname=gitkraken
-pkgrel=1
+pkgrel=2
 pkgver=2.5.0
+nodegitver=0.18.3
+electronver=1.2.8
 pkgdesc="The intuitive, fast, and beautiful cross-platform Git client."
-url="http://www.gitkraken.com/"
+url="https://www.gitkraken.com/"
```

Comment:
My fault, I thought It's the other way around. Reading diff is not for everyone!